sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -232,7 +232,44 @@ public void queryPlanDiagnostics() {
         }
     }
 
-    @Test(groups = {"simple"})
+    @Test(groups = {"simple"}, timeOut = TIMEOUT)
+    public void queryMetrics() {
```

Comment:
please use `dataProvider` for providing different query params.

this will help in the reliability for the test: less chance for test to timeout, better visiblity on failure on what exactly failed.

Reply:
done